hooks/pre/29-el7_upgrade_postgresql.rb

```diff
@@ -0,0 +1,4 @@
+# On EL7 the SCL override should be used instead
+if !app_value(:noop) && el7
```

Comment:
```suggestion
if !app_value(:noop) && el7?
```

Reply:
Bleh, I should really get in the habit of writing tests for hooks.

Reply:
For now I just fixed it. It would require a fair bit of stubbing to get things to work.